alternator-test/test_describe_endpoints.py

```diff
@@ -34,14 +34,16 @@ def test_describe_endpoints(dynamodb):
         # send it another describe_endpoints() request ;-) Note that the
         # address does not include the "http://" or "https://" prefix, and
         # we need to choose one manually.
-        url = "http://" + address
+        prefix = "https://" if request.config.getoption('https') else "http://"
```

Comment:
This approach - of deciding https vs http depending on the "--https" works, but I wonder if that's what we should really do. I think it would be better to look at dynamodb's existing endpoint (I hope there's an API for that) and see if it starts with http or https. This will also allow us to use https when communicating with Amazon if we used https (as I think we do) for the original connection.